### DISCUSSION TITLE
Missing classes in bootstrap-rtl.min.css

### DISCUSSION THREAD

**Discussion by ranle** [twbs/bootstrap#39596] [Q&A]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

`.bg-*` and `.rounded-*` classes are missing.
Probably more classes as well, but haven't tried them all.

### Motivation and context

I'm building a website builder SaaS.
Customers can choose to set their website LTR or RTL.
I'm using `.bg-light` and `.rounded-5` in the LTR version and it works.
For the same website templates, when a user changes their settings to RTL, these two are not set in the css for some reason.

**Comment by louismaximepiton** [comment]
Hey,

I just transformed this issue into a conversation.
I think this is not Bootstrap related since this small Codepen shows that `.rounded` and `.bg-primary` are working: https://codepen.io/louismaximepiton/pen/xxBrmPw

Can you provide any Codepen, StackBlitz or reproducible example ?
Are you using the good CSS URLs `bootstrap.rtl.min.css` ?

**Reply by julien-deramond** [reply]
I confirm that both classes are present in https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.rtl.min.css. The issue doesn't come from Bootstrap IMHO.

**Comment by Khuthaily** [comment]
I built CarryLinks as a Saas that supports _ltr-_ ([https://carrylinks.com](https://carrylinks.com)) as wells an _rtl-_ ([https://ar.carrylinks.com](https://ar.carrylinks.com)). I did not have any issues with `.bg-*` and `.rounded-*` classes.
